CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 ### Added
+- Added pylint `unspecified-encoding` and `missing-function-docstring` and ignored opensearchpy for lints (([#643](https://github.com/opensearch-project/opensearch-py/pull/643)))
```

Comment:
What does "ignored opensearchpy for lints" mean? Remove it?

Reply:
I think it's important to talk about https://github.com/macohen/opensearch-py/blob/9963f01cbfaa3841b801e7d2b0d5126ea444fd9b/noxfile.py#L122 or ignore it or you tell me I need to figure this out before it can be merged. pylint for generated code seemed challenging and I at least wanted to get the other things right first.

Reply:
Will https://github.com/opensearch-project/opensearch-py/issues/645 solve this?

Reply:
not sure. I'll look into the generated code on the next round. My guess is that the code generator code will have very specific needs, maybe requiring line by line instructions. will look..

Reply:
in the last PR I added back the original four lints for opensearchpy and opened a new issue to add the rest: #647